### PR TITLE
featureFix/ROC-2054-enter-pay-by-set-date

### PR DESCRIPTION
**Pull request by rattur**
### JIRA link ###

https://tools.hmcts.net/jira/browse/ROC-2054

### Change description ###

Change route and test case to redirect to check and send page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/features/ccj/routes/pay-by-set-date.ts [hunk 1]
```diff
@@ -30,6 +30,6 @@ export default express.Router()
         const { externalId } = req.params
         user.ccjDraft.payBySetDate = form.model
         await DraftCCJService.save(res, next)
-        res.redirect(Paths.repaymentPlanPage.uri.replace(':externalId', externalId))
+        res.redirect(Paths.checkAndSendPage.uri.replace(':externalId', externalId))
```
Comment (timja-kainos) [inline src/main/features/ccj/routes/pay-by-set-date.ts]: this should be 
```
Paths.checkAndSendPage.evaluateUri({externalId, externalId})
```

File: src/test/features/ccj/routes/pay-by-set-date.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ const externalId = sampleClaimObj.externalId
 
 const cookieName: string = config.get<string>('session.cookieName')
 const payBySetDatePage = Paths.payBySetDatePage.uri.replace(':externalId', externalId)
-const repaymentPlanPage = Paths.repaymentPlanPage.uri.replace(':externalId', externalId)
+const checkAndSendPage = Paths.checkAndSendPage.uri.replace(':externalId', externalId)
```
Comment (timja-kainos) [inline src/test/features/ccj/routes/pay-by-set-date.ts]: same here

Comment (rattur) [reply src/test/features/ccj/routes/pay-by-set-date.ts]: done

File: src/main/features/ccj/routes/pay-by-set-date.ts (hunk 1 above)
Comment (rattur) [reply src/main/features/ccj/routes/pay-by-set-date.ts]: done

**Pull request merged**